source/reference/versioning.txt

```diff
@@ -39,35 +39,26 @@ asked to upgrade in order to receive support.
 MongoDB Server Version Compatibility and Support
 ------------------------------------------------
 
-{+c2c-product-name+} supports the following versions of MongoDB Server:
+.. include:: /includes/fact-minimum-server-version-support.rst
```

Comment:
We should also have this bullet (used to be in the limitations section) under the second sentence here:

The minimum supported [Feature Compatibility Version](https://www.mongodb.com/docs/master/reference/command/setFeatureCompatibilityVersion/#mongodb-dbcommand-dbcmd.setFeatureCompatibilityVersion) is 6.0.